### DISCUSSION TITLE
Column Grouping table

### DISCUSSION THREAD

**Discussion by gopusrikanth04** [TanStack/table#3668] [General]
Hello All, 
 
   I have an requirement column grouping table in react-table. expanding and hiding the columns by clicking the column 
   
  
![column grouping](https://user-images.githubusercontent.com/47174113/151559413-404e76c8-f491-4acf-b329-efc4b7226461.png)

 How to solve his. any help
 
 Thank you

**Comment by sidgangs99** [comment]
Did you get any solution since the questions seems old. I have the same requirement.
